src/subgraph/types.ts

```diff
@@ -10,13 +10,18 @@ export interface DomainIdDto {
   id: string;
 }
 
+export interface RegistrarContractDto {
```

Comment:
what's the benefit of doing this as a single variable interface vs. just defining `contract?: string;` in the below `DomainDto` interface?

Reply:
Because the type is:
```
{
  id: string;
 }
 ```
 and not `string`

Reply:
And the Dto may change from just being a single variable of a string, to something else.